desktop/e2e/run.js

```diff
@@ -0,0 +1,144 @@
+#!/usr/bin/env node
```

Comment:
Test runner script and test files are unchanged from source in `wp-desktop` repo, save for updating `PROJECT_DIR` and `E2E_DIR` to point to the update folder location.